.github/workflows/turborepo-library-release.yml

```diff
@@ -38,6 +38,29 @@ jobs:
               npm i -g pnpm@8.9.0
             setup: |
               pnpm install
+          - host: ubuntu-latest
+            container: ghcr.io/napi-rs/napi-rs/nodejs-rust:stable-2023-09-17-alpine
+            install: |
+              apk add libc6-compat curl
+            setup: |
+              pnpm install
+            target: "x86_64-unknown-linux-musl"
+          - host: ubuntu-latest
+            container: ghcr.io/napi-rs/napi-rs/nodejs-rust:stable-2023-09-17-alpine
+            install: |
+              apk add libc6-compat curl
+              echo /root/.cargo/bin >> ${GITHUB_PATH}
+              echo /usr/local/cargo/bin/rustup >> ${GITHUB_PATH}
+              echo /aarch64-linux-musl-cross/bin >> ${GITHUB_PATH}
+              export PATH=/aarch64-linux-musl-cross/bin:/usr/local/cargo/bin/rustup:/root/.cargo/bin:${PATH}
+            setup: |
+              export PATH=/aarch64-linux-musl-cross/bin:/usr/local/cargo/bin/rustup:/root/.cargo/bin:${PATH}
```

Comment:
Huh, still strange it's required to fix up the PATH differences between local / github actions

Reply:
Biggest struggle was PATH differences. It's possible I'm doing something wrong, but as best I can tell, the PATH when running via GH Actions is different from the PATH running the same docker image on my machine. So I went for overkill in making sure I could run the binaries I needed.